src/cpu/core_dynrec/decoder_basic.h

```diff
@@ -720,7 +730,8 @@ bool DRC_CALL_CONV mem_readw_checked_drc(PhysPt address) {
 	if ((address & 0xfff)<0xfff) {
 		HostPt tlb_addr=get_tlb_read(address);
 		if (tlb_addr) {
-			*((Bit16u*)(&core_dynrec.readdata))=host_readw(tlb_addr+address);
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the host_* functions do).